### PR TITLE
suggest setting up continuous integration before continuous deployment

### PR DESCRIPTION
**Pull request by afeld**
/cc @rdgsa

### CODE REVIEW

File: content/docs/apps/continuous-deployment.md [hunk 1]
```diff
@@ -8,7 +8,12 @@ title: Continuous deployment
 Setting up continuous deployment allows you to automatically upload your
 changes to your desired environment.
 
-Before setting this up, go through the [Production Ready Guide]({{< relref "production-ready.md" >}}) and make sure you're following those core best practices.
+Before setting this up, go through
+
+1. Setting up continuous integration
+    * This will protect you from deploying a broken application.
+    * You can use the same service for continuous integration+deployment — see list [below]({{< relref "#continuous-integration-services" >}}) for suggestions.
+1. The [Production Ready Guide]({{< relref "production-ready.md" >}})
```
Comment (LinuxBozo) [inline content/docs/apps/continuous-deployment.md]: I would reverse the order of those.. Production Ready gets you ready for CI, by pointing out using a manifest, etc.

**Review by LinuxBozo — CHANGES_REQUESTED** [review]

File: content/docs/apps/continuous-deployment.md (hunk 1 above)
Comment (afeld) [reply content/docs/apps/continuous-deployment.md]: Makes total sense! Updated.

**Review by LinuxBozo — APPROVED** [review]

**Comment by brittag** [comment]
@LinuxBozo @afeld I adjusted this intro to give a bit more context - does it still look right? I especially wanted to narrow the reference to the production-ready guide to avoid a loop, since one of the production-ready guide recommendations is to use continuous deployment. :)

screen shot 2017-01-13 at 10 40 30 am

This commit also clarifies the Travis instructions a little.

File: content/docs/apps/continuous-deployment.md [hunk 2]
```diff
@@ -64,7 +69,13 @@ To set up any of these services, your application needs a `manifest.yml` file.
 
 ### Travis
 
-See [the Travis documentation](http://docs.travis-ci.com/user/deployment/cloudfoundry/), using `api: https://api.cloud.gov` for East/West and `api: https://api.fr.cloud.gov` for GovCloud. You must encrypt the password, and you must [**escape any symbol characters in the password**](https://docs.travis-ci.com/user/encryption-keys#Note-on-escaping-certain-symbols), to prevent possible situations where Travis could dump an error message that contains passwords or environment variables.
+See [the Travis documentation](http://docs.travis-ci.com/user/deployment/cloudfoundry/). 
+
+On East/West: use `api: https://api.cloud.gov`
+
+On GovCloud: use `api: https://api.fr.cloud.gov`
```
Comment (afeld) [inline content/docs/apps/continuous-deployment.md]: I would suggest we use our environment blocks, for consistency.

**Comment by afeld** [comment]
Just the one thing - good for me otherwise!

File: content/docs/apps/continuous-deployment.md (hunk 2 above)
Comment (brittag) [reply content/docs/apps/continuous-deployment.md]: Done! Also added info about this for the other suggested services.

**Pull request merged**